projects/packages/my-jetpack/_inc/data/types.ts

```diff
@@ -1,74 +1,3 @@
-export type Purchase = {
```

Comment:
We don't seem to be using this type. Thought we might remove it until we need it? Probably whenever we convert all of My Jetpack to TypeScript

Reply:
I think we can keep it for now, and decide during the project cleanup phase if we have any plan of using it. Definitely not needed to remove it in this PR, but I don't mind if you think that's a good idea.

Reply:
Talked in slack and decided to remove this since we aren't converting the components to TS yet